### PR TITLE
support choose a conversation style of BingAI

### PR DESCRIPTION
**Pull request by likev**
choose a conversation style

### CODE REVIEW

File: src/BingAIClient.js [hunk 1]
```diff
@@ -150,26 +151,35 @@ export default class BingAIClient {
...
+                        "cricinfov2",
+                        "dv3sugg"
                     ],
                     sliceIds: [
-                        '222dtappid',
-                        '216dloffstream',
-                        '225cricinfos0',
+                        "222dtappid",
+                        "225cricinfo",
+                        "224locals0"
```
Comment (waylaidwanderer) [inline src/BingAIClient.js:182]: Can you fix these quotes?

File: src/BingAIClient.js [hunk 2]
```diff
@@ -150,26 +151,35 @@ export default class BingAIClient {
 
         const ws = await this.createWebSocketConnection();
 
+        let toneOption;
+        if (toneStyle === 'creative') {
+            toneOption = 'h3imaginative';
+        }else if (toneStyle === 'precise') {
+            toneOption = 'h3precise';
+        }else {
```
Comment (waylaidwanderer) [inline src/BingAIClient.js:159]:
```suggestion
        } else if (toneStyle === 'precise') {
            toneOption = 'h3precise';
        } else {
```

File: src/BingAIClient.js [hunk 3]
```diff
@@ -121,6 +121,7 @@ export default class BingAIClient {
         opts = {},
     ) {
         let {
+            toneStyle = 'balanced',//or creative, precise
```
Comment (waylaidwanderer) [inline src/BingAIClient.js:124]:
```suggestion
            toneStyle = 'balanced', //or creative, precise
```

File: demos/use-bing-client.js [hunk 4]
```diff
@@ -21,6 +21,7 @@ let response = await bingAIClient.sendMessage('Write a short poem about cats', {
 console.log(response);
 
 response = await bingAIClient.sendMessage('Now write it in French', {
+    toneStyle: 'balanced',//or creative, precise
```
Comment (waylaidwanderer) [inline demos/use-bing-client.js:24]:
```suggestion
    toneStyle: 'balanced', //or creative, precise
```

**Comment by waylaidwanderer** [comment]
@likev looks great; thanks so much! Left a few comments regarding changes you should make.

File: src/BingAIClient.js (hunk 1 above)
Comment (likev) [reply src/BingAIClient.js:182]: quotes are fixed

**Comment by waylaidwanderer** [comment]
Thanks so much!

**Pull request merged**